tools/createComponent.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs-extra')
+const path = require('path')
+
+const {index, style, spec} = require('./lib/templates')
+
+if (process.argv.length <= 2) return
+
+const componentName = process.argv[2]
+const getFolderName = (name) => {
+  return name[0].toLowerCase() + name.slice(1)
+}
+
+const defaultDir = path.join(__dirname, '../src')
+const parentDir = process.argv.length === 4
+  ? path.join(defaultDir, 'features', process.argv[3])
+  : path.join(defaultDir, 'components')
+const componentDir = path.join(parentDir, getFolderName(componentName))
+
+const writeItems = [
+  {type: 'dir', path: 'assets'},
```

Comment:
This will be more exception then a rule, because most of the assets will come from Icon component